README.md

```diff
@@ -64,7 +64,7 @@ Download
 the [latest version](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar).
 
 This package includes the instrumentation agent as well as
-instrumentations for all supported libraries and all available data exporters.
+instrumentation for all supported libraries and all available data exporters.
```

Comment:
@theletterf could you review this change

Reply:
IntelliJ complained about the grammar here, and I changed it. I have other similar changes uncommitted locally. Please advise on how to proceed.